packages/commonwealth/client/scripts/views/components/sidebar/AccountConnectionIndicator/AccountConnectionIndicator.tsx

```diff
@@ -1,102 +1,53 @@
 import { useFlag } from 'client/scripts/hooks/useFlag';
-import { saveToClipboard } from 'client/scripts/utils/clipboard';
-import clsx from 'clsx';
+import AddressInfo from 'client/scripts/models/AddressInfo';
+import NewProfile from 'client/scripts/models/NewProfile';
+import { AuthModalType } from 'client/scripts/views/modals/AuthModal';
 import React from 'react';
 import { useInviteLinkModal } from 'state/ui/modals';
-import useUserStore from 'state/ui/user';
 import useJoinCommunity from 'views/components/SublayoutHeader/useJoinCommunity';
-import { CWText } from 'views/components/component_kit/cw_text';
 import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
-import { CWIdentificationTag } from 'views/components/component_kit/new_designs/CWIdentificationTag';
-import { handleMouseEnter, handleMouseLeave } from 'views/menus/utils';
-import useAuthentication from '../../../modals/AuthModal/useAuthentication';
 import { SharePopover } from '../../SharePopover';
-import CWIconButton from '../../component_kit/new_designs/CWIconButton';
-import { CWTooltip } from '../../component_kit/new_designs/CWTooltip';
+import { AddressList } from '../CommunitySection/AddressList';
 import './AccountConnectionIndicator.scss';
 
 interface AccountConnectionIndicatorProps {
   connected: boolean;
   address: string;
+  addresses: AddressInfo[] | undefined;
+  profile: NewProfile | undefined;
+  refreshProfiles: () => void;
+  onAuthModalOpen: (modalType?: AuthModalType) => void;
 }
 
 const AccountConnectionIndicator = ({
   connected,
   address,
+  profile,
+  addresses,
+  refreshProfiles,
+  onAuthModalOpen,
 }: AccountConnectionIndicatorProps) => {
   const { handleJoinCommunity, JoinCommunityModals } = useJoinCommunity();
   const referralsEnabled = useFlag('referrals');
   const { setIsInviteLinkModalOpen } = useInviteLinkModal();
 
-  const userData = useUserStore();
-  const hasMagic = userData.addresses?.[0]?.walletId === WalletId.Magic;
-
-  const { openMagicWallet } = useAuthentication({});
+  if (!profile) {
+    return null;
+  }
 
   return (
     <>
       <div className="AccountConnectionIndicator">
         {connected && (
           <div className="status-address">
-            <CWText fontWeight="medium" type="caption" className="status-text">
-              {connected ? 'Connected' : 'Not connected'}
-            </CWText>
             <div className="status-row">
-              <div className={clsx('status-light', { connected })} />
-              <CWIdentificationTag address={address} />
-              <CWTooltip
-                placement="top"
-                content="address copied!"
-                renderTrigger={(handleInteraction, isTooltipOpen) => {
-                  return (
-                    <CWIconButton
-                      iconName="copySimple"
-                      onClick={(event) => {
-                        saveToClipboard(address).catch(console.error);
```

Comment:
Are we completely getting rid of the copy functionality, or is this being put back in in another PR?

Reply:
We are keeping it